packages/app/src/device/actions/base.action.test.ts

```diff
@@ -136,22 +136,22 @@ describe("Action: setInitState", () => {
   })
 })
 
-describe("Action: setAgreementStatus", () => {
-  test("fire action with `AgreementStatus` type with provided payload", () => {
-    mockStore.dispatch(setAgreementStatus(true))
+describe("Action: setOnboardingStatus", () => {
```

Comment:
What's the point of testing if action dispatch works? That's functionality provided by a third-party package, so I think we shouldn't test that. If you want to ensure that the type of payload is correct it's already done by typescript.

Reply:
I think that these tests can be of value in the context of regression checking when you make changes, for example. Typescript won't catch the action change only the type of it.

Reply:
I disagree, it's still checking typescript. I'll approve PR to not block task

Reply:
discussed asynchronously. I'm going to make a quick fix. And add the change as a topic on the front-end meetup

Reply:
But I can't fix it right away (if I don't have to). Will complete this task in the next tech task sprint https://appnroll.atlassian.net/browse/CP-2150

Reply:
For me it looks like this:
- dispatching action on store
- verify already run actions

It's testing of mechanism running actions on store.